asab/library/providers/git.py

```diff
@@ -72,6 +72,8 @@ def __init__(self, library, path):
 		self.ProactorService = self.App.get_service("asab.ProactorService")
 		self.PullLock = False
 
+		self.SubscribedPath = None
```

Comment:
It should be a set/list ... and I suggest to consider having this in `abc`.

Reply:
Considering having this in abc - there is a different structure in the FileProvider... I'll think a bit about whether I could unify it somehow.